.wercker.yml

```diff
@@ -12,6 +12,7 @@ build:
                 conda config --set show_channel_urls True
                 conda config --add channels conda-forge
                 source activate root
+                echo "conda-build 1.*" >> /opt/conda/conda-meta/pinned
```

Comment:
Had to pin here explicitly as we were running into build issues with PR ( https://github.com/jakirkham/docker_centos_drmaa_conda/pull/22 ). Doesn't seem that repeating the pinning should cause any issues based on local testing.